kernel/enter.c

```diff
@@ -820,6 +939,104 @@ int myst_enter_kernel(myst_kernel_args_t* args)
     if (__myst_kernel_args.perf)
         _print_boottime();
 
+    // open the appenv.json
+    char* data;
+    size_t size;
+    char* path = "appenv.json";
```

Comment:
If this is a hard-coded string, can you use `#define`?

Reply:
added this, thank you